src/crimson/os/seastore/random_block_manager/nvme_manager.cc

```diff
@@ -62,28 +57,28 @@ NVMeManager::mkfs_ertr::future<> NVMeManager::initialize_blk_alloc_area() {
 
   return rbm_sync_block_bitmap(b_block,
     super.start_alloc_area / super.block_size
-  ).safe_then([this, b_block, start] () mutable {
+  ).safe_then([this, b_block, start, FNAME] () mutable {
```

Comment:
our convention is `[...]() mutable {` or `[...] {`.

`mutable` is also not needed here.